resources/dicts/events/ceremonies/ceremony_traits.json

```diff
@@ -76,7 +101,13 @@
         "phlegmatism",
```

Comment:
This should probably be removed (i think i've seen it suggested before?), along with the other instances of it, since it's a pretty obscure word and is too close to "phlegm" (meaning snot/spit)

Reply:
sure yeah that makes sense, lemme grab a different word